src/Microsoft.Health.Fhir.Core/Features/Definition/SearchParametersHashUpdatedHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using MediatR;
+using Microsoft.Health.Fhir.Core.Messages.Search;
+
+namespace Microsoft.Health.Fhir.Core.Features.Definition
+{
+    public class SearchParametersHashUpdatedHandler : INotificationHandler<SearchParametersHashUpdated>
```

Comment:
nit: This class seems out of place in this Features.Definition namespace. Would it be better to move it to a location where other handlers are present?

Reply:
I do see the point, we are not entirely consistent with the location of handlers.  Some other features, Compartment, Conformance, Export also contain Handlers in the feature namespace, I believe with the intent of keeping all classes relevant to a feature in one location.  And as you said, some handlers are collected together.  I do not feel strongly either way.